integrations/http/save.go

```diff
@@ -55,33 +59,7 @@ func (h handler) handleAuth(w http.ResponseWriter, r *http.Request) {
 		auth = "Bearer " + bearer
 	}
 
-	// Save a new connection, and return to the user an autokitteh connection token.
-	connToken, err := h.createConnection(auth)
-	if err != nil {
-		l.Warn("Failed to save new connection secrets", zap.Error(err))
-		e := "Connection saving error: " + err.Error()
-		u := fmt.Sprintf("%serror.html?error=%s", uiPath, url.QueryEscape(e))
-		http.Redirect(w, r, u, http.StatusFound)
-		return
-	}
-
-	// Redirect the user to a success page: give them the connection token.
-	l.Debug("Saved new autokitteh connection")
-	u := fmt.Sprintf("%ssuccess.html?token=%s", uiPath, connToken)
-	http.Redirect(w, r, u, http.StatusFound)
-}
+	initData := sdktypes.NewVars().Set(authVar, auth, true)
 
-func (h handler) createConnection(auth string) (string, error) {
-	token, err := h.secrets.Create(context.Background(), h.scope,
-		// Connection token --> authorization HTTP header.
-		map[string]string{
-			"authorization": auth,
-		},
-		// List of all connection tokens.
-		"tokens",
-	)
-	if err != nil {
-		return "", err
-	}
-	return token, nil
+	sdkintegrations.FinalizeConnectionInit(w, r, IntegrationID, initData)
```

Comment:
could u pplz help to understand where/how auth is stored?

Reply:
this redirects to https://github.com/autokitteh/autokitteh/blob/489ce1df324f2f11aa5735ca7e7b935500fb9ae6/internal/backend/dashboardsvc/connections.go#L179, which stores the connection vars.